### PR TITLE
Issue 693: Video component is flipped horizontally

### PR DESCRIPTION
**Pull request by dawoodkhan82**
# Description

Please include: 
* relevant motivation
* a summary of the change 
* which issue is fixed. 
* any additional dependencies that are required for this change.

Fixes: # (issue)
https://github.com/gradio-app/gradio/issues/693

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by pngwn** [comment]
Do we want to keep the webcam flipped but prevent this from affecting other video elements? If so we can just remove the `global` attribute on line 70. By default svelte will scope CSS to the component but global removes that behaviour.

File: ui/packages/app/src/components/utils/Webcam.svelte [hunk 1]
```diff
@@ -66,10 +66,3 @@
 		/>
 	</button>
 </div>
-
-<style lang="postcss" global>
-	video {
-		-webkit-transform: scaleX(-1);
-		transform: scaleX(-1);
-	}
-</style>
```
Comment (abidlabs) [inline ui/packages/app/src/components/utils/Webcam.svelte:75]: I thought we needed these lines because the webcam was flipped (#627)?

**Comment by dawoodkhan82** [comment]
> @dawoodkhan82 Do we want to keep the webcam flipped but prevent this from affecting other video elements? If so we can just remove the `global` attribute on line 70. By default svelte will scope CSS to the component but global removes that behaviour.

done

File: ui/packages/app/src/components/utils/Webcam.svelte (hunk 1 above)
Comment (dawoodkhan82) [reply ui/packages/app/src/components/utils/Webcam.svelte]: added back, but removed global, as per @pngwn suggestion

**Review by pngwn — APPROVED** [review]

**Pull request merged**